<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>421</width>
     <height>251</height>
    </rect>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>310</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font: 18pt &quot;Yu Gothic UI Semilight&quot;;
	background-image: url(:/Image/UIelement/bui.png);
	background-color:transparent;
	background-repeat: no-repeat;
}
QPushButton:hover{
	background-position: 10px 0px;
}
QPushButton:pressed{
	font-size:30pt;
	font-style:oblique;
	font-variant:small-caps;
	border-radius:25px;
	border-style:inset;
	
}</string>
    </property>
    <property name="text">
     <string>点击生成</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font: 8pt &quot;Yu Gothic UI Semilight&quot;;
	background-image: url(:/Image/UIelement/Exc.png);
	background-color:transparent;
	background-repeat: no-repeat;
}
QPushButton:hover{
	background-position: 10px 0px;
	border-radius:15px;
}
QPushButton:pressed{
	font-size:10pt;
	font-style:oblique;
	font-variant:small-caps;
	border-bottom-left-radius:15px;
	border-top-left-radius:15px;
	border-style:inset;
	background-color:#cccccccc
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>361</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Image/UIelement/background.png);
border-radius:25px;
border-top-left-radius:50px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>351</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(85, 170, 255,80);
border-radius:50px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;
background-color: rgba(255, 255, 255,100);
</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>pushButton</zorder>
   <zorder>closeButton</zorder>
   <zorder>lineEdit</zorder>
  </widget>
 </widget>
 <resources>
  <include location="UIResource.qrc"/>
 </resources>
 <connections/>
</ui>
